utils/imodels-client-test-utils/src/test-context-providers/imodel/TestIModelFileProvider.ts

```diff
@@ -26,20 +27,21 @@ interface ChangesetDescriptorFileItem extends TestChangesetFile {
   fileName: string;
 }
 
+@injectable()
 export class TestIModelFileProvider {
-  private static readonly _iModelDataRootPath = `${__dirname}/../../../assets/test-imodel`;
-  private static _baselineFile: TestIModelBaselineFile;
-  private static _changesetFiles: TestChangesetFile[];
+  private readonly _iModelDataRootPath = `${__dirname}/../../assets/test-imodel`;
+  private _baselineFile: TestIModelBaselineFile | undefined;
```

Comment:
Why not nullable?

Reply:
Same reason: https://github.com/iTwin/imodels-clients/pull/65#discussion_r772120802